src/components/user-profile.html

```diff
@@ -0,0 +1,135 @@
+                margin-top: 25px;
+                padding-bottom: 25px;
+            }
+
+            form {
+                border: 1 px black solid;
+                border-radius: 5px;
+                background-color: #f2f2f2;
+                padding: 20px;
+                width: 50%;
+                margin: auto;
+            }
+
+            .button {
+                background-color: #87CEFA;
+                border: none;
+                color: white;
+                padding: 5px 12px;
+                text-align: center;
+                text-decoration: none;
+                display: block;
+                margin: auto;
+                font-size: 16px;
+                margin-top: 25px;
+            }
+
+            button:hover {
+                background-color: rgb(196, 226, 245);
+            }
+
+            table.center {
+                margin-right: auto;
+                margin-left: auto;
+            }
+
+            th,
+            td {
+                text-align: left;
+                padding: 16px;
+            }
+
+            tr:nth-child(odd) {
+                background-color: #f2f2f2;
+            }
+        </style>
+
+        <div class="container">
+
+            <button class="button" on-click="expandCollapse">Expand/Collapse Profile</button>
+
+            <div class="form" id="userProfileCard">
+
+                <div>
+
+                    <div class="div-center">
+
+                        <h3 class="div-center">User Profile</h3>
+
+                        <table class="center">
+                            <tr>
+                                <td>ID:</td>
+                                <td>12345</td>
+                            </tr>
+                            <tr>
+                                <td>First Name:</td>
+                                <td>Jane</td>
+                            </tr>
+                            <tr>
+                                <td>Last Name:</td>
+                                <td>Doe</td>
+                            </tr>
+                            <tr>
+                                <td>Phone:</td>
+                                <td>111-111-1111</td>
+                            </tr>
+                            <tr>
+                                <td>Email:</td>
+                                <td>jane.doe@test.com</td>
+                            </tr>
+                        </table>
+
+                    </div>
+                </div>
+            </div>
+        </div>
+
+    </template>
+
+    <script type="module">
+
+        import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
+        
+        class UserProfile extends PolymerElement {
+
+            static get is() { return 'user-profile'; }
+                            
+            expandCollapse() {
+
+                let userCard = this.$.userProfileCard;
+
+                if (userCard.style.display === "block") {
```

Comment:
This is failing on the first click because there is no `style` attribute that sets its value to "block". So, the first click treats it as if it was hidden and adds `style="display: block"` to the element attributes. Consider using [window.getComputedStyle](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle) to make the calculation, or adding an initial style attribute to the element in HTML.